bin/migrate-oats-data/applications/submissions/data_insert/insert_query.py

```diff
@@ -272,8 +319,8 @@ def get_insert_query_for_soil():
                         %(max_remove_depth)s,
                         %(remove_duration)s,
                         %(fill_duration)s,
-                        %(remove_duration_unit)s,
-                        %(fill_duration_unit)s,
+                        %(duration_unit)s,
+                        %(duration_unit)s,
```

Comment:
should one of this be a duration_unit and the other fill_duration_unit?

Reply:
It is mapped to the same value of 'months' so it will be the same